### PR TITLE
Change button design in screen "Risiko-Ermittlung" (EXPOSUREAPP-5451)

### PR DESCRIPTION
**Pull request by LukasLechnerDev**
Before: 

image

Designs: 

image

After: 

image

### CODE REVIEW

**Comment by ralfgehrer** [comment]
Screenshot 2021-03-04 at 14 03 22

Touch highlighting is off

File: Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml [hunk 1]
```diff
@@ -92,35 +92,54 @@
                     app:statusText="@{settingsTracingState.getTracingStatusText(context)}"
                     app:subtitle="@{@string/settings_tracing_title}" />
 
-                <androidx.appcompat.widget.LinearLayoutCompat
-                    android:id="@+id/settings_interoperability_row"
-                    style="@style/row"
-                    android:layout_width="match_parent"
+                <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml:95]: Although the performance was ok on my Pixel, this introduced at third-level nested Constraint Layout

**Review by AlexanderAlferov — DISMISSED** [review]
Tested on Samsung S8. No performance issues found. Looks good 🚀

File: Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml (hunk 1 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml:95]: Thanks! - Yes, I am aware of this. However, I haven't found a better solution without refactoring the whole layout. Do you have an idea?

File: Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml [hunk 2]
```diff
@@ -92,35 +92,54 @@
...
-                    android:orientation="vertical"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
+                    android:paddingTop="@dimen/spacing_tiny"
+                    android:paddingBottom="@dimen/spacing_tiny"
+                    android:background="?android:selectableItemBackground"
+                    app:layout_constraintStart_toStartOf="@id/guideline_start"
                     app:layout_constraintTop_toBottomOf="@id/settings_tracing_switch_row">
 
+                    <ImageView
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml:105]: Please use ImageAppCompat

File: Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml [hunk 3]
```diff
@@ -92,35 +92,54 @@
...
+                        android:id="@+id/flag_eu"
+                        android:layout_width="26dp"
+                        android:layout_height="26dp"
+                        android:src="@drawable/ic_country_eu"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml:109]: srcCompat

File: Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml [hunk 4]
```diff
@@ -92,35 +92,54 @@
...
+                        android:layout_width="0dp"
                         android:layout_height="wrap_content"
-                        android:text="@string/settings_interoperability_title" />
+                        android:layout_marginStart="@dimen/spacing_tiny"
+                        android:text="@string/settings_interoperability_title"
+                        app:layout_constraintBottom_toBottomOf="@id/flag_eu"
+                        app:layout_constraintStart_toEndOf="@id/flag_eu"
+                        app:layout_constraintTop_toTopOf="@id/flag_eu" />
+
+                    <ImageView
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml:124]: same here

File: Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml [hunk 5]
```diff
@@ -92,35 +92,54 @@
...
+                        android:id="@+id/flag_ch"
+                        android:layout_width="0dp"
+                        android:layout_height="22dp"
+                        android:src="@drawable/ic_country_ch"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml:128]: srcCompat

**Comment by LukasLechnerDev** [comment]
@ralfgehrer Good Catch! I fixed the touch area in the commit above 👆

**Comment by LukasLechnerDev** [comment]
@mtwalli I replaced ImageView with AppCompatImageView.

**Review by mtwalli — DISMISSED** [review]

**Review by ralfgehrer — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]

**Pull request merged**